Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -602,8 +643,10 @@ function ViewPanelAmenities( data:table )
     end
 
     -- Districts
-	data.AmenitiesFromDistricts = data.AmenitiesFromDistricts or 0;
-    CQUI_BuildAmenityBubbleInstance("ICON_DISTRICT_CITY_CENTER", data.AmenitiesFromDistricts, "LOC_PEDIA_DISTRICTS_TITLE");
+    data.AmenitiesFromDistricts = data.AmenitiesFromDistricts or 0;
+    if data.AmenitiesFromDistricts > 0 then
```

Comment:
The unmodified Firaxis version of this file only shows AmenitiesFromDistricts if the value is greater than zero

Reply:
This was a purposful change. Please don't add this condition.

Reply:
ok